plugin/pkg/scheduler/api/v1/register.go

```diff
@@ -26,7 +27,16 @@ import (
 var SchemeGroupVersion = unversioned.GroupVersion{Group: "", Version: "v1"}
 
 func init() {
-	api.Scheme.AddKnownTypes(SchemeGroupVersion,
+	addKnownTypes(api.Scheme)
```

Comment:
Ironically this makes it harder to understand what is happening here (we add types on init, create a scheme builder that calls a local method as well, then define a public var that doesn't point to the local method except via indirection.  I understand it, it's just full of twisty packages.